scripts/build/release.sh

```diff
@@ -168,19 +168,19 @@ touch CHANGES_NEXT_RELEASE
 # dev release sets 'latest' and not 'X.Y.Z-next"
 if [ "$BROKER_RELEASE" != "dev" ]
 then
-  sed "s/(https:\/\/readthedocs.org\/projects\/fiware-orion\/badge\/?version=latest)](http:\/\/fiware-orion.readthedocs.io\/en\/latest\/?badge=latest)/(https:\/\/readthedocs.org\/projects\/fiware-orion\/badge\/?version=$NEW_VERSION)](http:\/\/fiware-orion.readthedocs.io\/en\/$NEW_VERSION\/?badge=$NEW_VERSION)/" README.md > /tmp/README.md
+  sed "s/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.svg)](https:\/\/fiware-orion.rtfd.io)/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.$NEW_VERSION.svg)](https:\/\/fiware-orion.rtfd.io\/en\/$NEW_VERSION\/)/" README.md > /tmp/README.md
```

Comment:
Let's test the new sed command in a release scenario (i.e. `[ "$BROKER_RELEASE" != "dev" ]` evaluates to true). Let current version be 1.14.0-next and new version 1.15.0.

Input line (as it is in README.md):
```
[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion.svg)](https://fiware-orion.rtfd.io) 
```
Test:

```
$ export NEW_VERSION=1.15.0
$ export currentVersion=1.14.0-next
$ echo '[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion.svg)](https://fiware-orion.rtfd.io)' | sed "s/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.svg)](https:\/\/fiware-orion.rtfd.io)/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.$NEW_VERSION.svg)](https:\/\/fiware-orion.rtfd.io\/en\/$NEW_VERSION\/)/"
```

Result:

```
[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion.1.15.0.svg)](https://fiware-orion.rtfd.io/en/1.15.0/)
```

* https://img.shields.io/readthedocs/fiware-orion.1.15.0.svg, doesn't work (it shows "docs | inaccesible")
* https://fiware-orion.rtfd.io/en/1.15.0/ does work

Reply:
Fixed 18d31bc6 - it should be a`/` separator, not a `.` between the repo name and the version.

Reply:
I do confirm that this couple of seds work fine:

Relasing:

```
$ export NEW_VERSION=1.15.0
$ export currentVersion=1.14.0-next
$ echo '[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion.svg)](https://fiware-orion.rtfd.io)' |sed "s/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.svg)](https:\/\/fiware-orion.rtfd.io)/(https:\/\/img.shields.io\/readthedocs\/fiware-orion\/$NEW_VERSION.svg)](https:\/\/fiware-orion.rtfd.io\/en\/$NEW_VERSION\/)/"
[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion/1.15.0.svg)](https://fiware-orion.rtfd.io/en/1.15.0/)
```

Opening new development cycle:

```
$ export currentVersion=1.15.0
$ echo '[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion/1.15.0.svg)](https://fiware-orion.rtfd.io/en/1.15.0/)' | sed "s/(https:\/\/img.shields.io\/readthedocs\/fiware-orion\/$currentVersion.svg)](https:\/\/fiware-orion.rtfd.io\/en\/$currentVersion\/)/(https:\/\/img.shields.io\/readthedocs\/fiware-orion.svg)](https:\/\/fiware-orion.rtfd.io)/"
[![Documentation badge](https://img.shields.io/readthedocs/fiware-orion.svg)](https://fiware-orion.rtfd.io)
```